### ISSUE TITLE
LLaVa support

### ISSUE THREAD

**Issue by jhen0409** [mybigday/llama.rn#33]
llama.cpp includes LLaVa example (+clip.cpp), we could use it to provide vision support. We may implement it after #30 is done.

Also, it will be great if we could make an another package named clip.rn or react-native-clip, but currently I afraid we haven't more resources to maintain it, so just keep in mind.

**Comment by tctrautman** [comment]
Hey!  I just stumbled upon this project -- it's great to see the work you have done here, and thanks for open sourcing it.

I'm just curious -- any rough ETA or sense of when this will get worked on?  I completely understand if it's not a priority right now.
